app/domain/authentication/usecase/create_token.go

```diff
@@ -0,0 +1,51 @@
+package usecase
+
+import (
+	"github.com/golang-jwt/jwt"
+	"github.com/patriciapedrosaa/transfer-me/app/domain/entities"
+	"os"
+)
+
+func (a Authentication) CreateToken(login LoginInputs) (string, error) {
+	validUser, err := a.CheckLogin(login)
+	if !validUser {
+		return "", err
+	}
+
+	account, err := a.accountRepository.GetByCpf(login.CPF)
```

Comment:
`CheckLogin` will currently retrieve an account(which it's not its responsibility since this is not an account domain) and then checks if passwords match. After that, you retrieve the same account. Don't you think this information it's an input as well?